packages/fleet-agent/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,13 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -5,8 +5,8 @@
+   catalog.cattle.io/os: linux
+   catalog.cattle.io/release-name: fleet-agent
+ apiVersion: v2
+-appVersion: 0.3.4-rc3
++appVersion: 0.3.3
```

Comment:
I think these versions should still be 0.3.4. When the patch is applied, I believe you want it to be 0.3.400-rc01

Reply:
Good catch. Strange that they are flipped. Fixed now

Reply:
You still need to remove the `-rc3`. The version should just be `0.3.4` and the rc will be added by Arvind's scripts.